static-sample/js/create-load.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
```

Comment:
It's there temporarily so we don't lose your work while I build up the framework to do it in a more modern way. Unless you mean that this code isn't used?